recipes-support/collision-avoidance-library/coav-control_0.0.1.bb

```diff
@@ -24,6 +24,6 @@ do_install () {
     chrpath -d ${B}/tools/coav-control/coav-control
```

Comment:
Because coav-control's make install, which indeed deals with rpath, also installs the library and causes a conflict. Solving this would be nice, but low prio for now. Anyway, thanks for reminding me to open an issue in collision avoidance library's repo.